### PR TITLE
2020 14 uniparc results

### PR DESCRIPTION
**Pull request by aurel-l**
## Purpose
Search results page for UniParc [TRM-25177](https://www.ebi.ac.uk/panda/jira/browse/TRM-25177)

## Approach
 - Have a renderer for every column in the UniParc search results page
 - Added facets

## Testing
Snapshot for every column renderer + 2 edge cases tests

## Checklist
- [ ] My PR is scoped properly, and “does one thing only”
  - 😬 I also fixed an issue when switching namespaces when in search results page (before it would use stale data with new renderers)
- [x] I have reviewed my own code
- [x] I have checked that linting checks pass and type safety is respected
- [x] I have checked that tests pass and coverage has at least improved, and if not explained the reasons why

### CODE REVIEW

File: src/uniparc/config/UniParcColumnConfiguration.tsx [hunk 1]
```diff
@@ -92,20 +172,54 @@ UniParcColumnConfiguration.set(UniParcColumn.organismID, {
 
 UniParcColumnConfiguration.set(UniParcColumn.organism, {
   label: 'Organisms',
-  // TODO:
-  render: () => null,
+  /* TODO:
```
Comment (aurel-l) [inline src/uniparc/config/UniParcColumnConfiguration.tsx:175]: This data is missing from the payload

File: src/uniparc/config/UniParcColumnConfiguration.tsx [hunk 2]
```diff
@@ -58,6 +65,71 @@ export const UniParcColumnConfiguration = new Map<
   }
 >();
 
+/* NOTE:
```
Comment (xwatkins) [inline src/uniparc/config/UniParcColumnConfiguration.tsx:68]: Seems like a good idea - would move to shared utils though?

File: src/shared/components/results/ResultsContainer.tsx [hunk 3]
```diff
@@ -84,10 +84,11 @@ const Results: FC = () => {
   const initialApiUrl = getAPIQueryUrl({
     namespace,
     query,
-    columns,
+    columns: namespace === Namespace.uniparc ? undefined : columns,
```
Comment (xwatkins) [inline src/shared/components/results/ResultsContainer.tsx:87]: Not too keen on that but it's linked to the hack L81 (instead of using `[]` we should explicitly define the columns we want to retrieve for cards and for table for each namespace, and I guess "undefined" would be accepted). Maybe move this to L81 so all the hack is in the same place and add a comment here https://www.ebi.ac.uk/panda/jira/browse/TRM-25288

File: src/shared/components/results/ResultsView.tsx [hunk 4]
```diff
@@ -104,8 +109,10 @@ const ResultsView: FC<ResultsTableProps> = ({
   const initialApiUrl = getAPIQueryUrl({
     namespace,
     query,
-    columns,
+    columns: namespace === Namespace.uniparc ? undefined : columns,
```
Comment (xwatkins) [inline src/shared/components/results/ResultsView.tsx:112]: Same comment as above

**Review by xwatkins — APPROVED** [review]

File: src/uniparc/config/UniParcColumnConfiguration.tsx (hunk 2 above)
Comment (aurel-l) [reply src/uniparc/config/UniParcColumnConfiguration.tsx:68]: will do

File: src/shared/components/results/ResultsContainer.tsx (hunk 3 above)
Comment (aurel-l) [reply src/shared/components/results/ResultsContainer.tsx:87]: I can't really do that in l.81 as the value as it is in l.81 is required in l.180 and passed down to the result view component in order to render the correct columns. l.87 is the only place where I need to override it

Comment (xwatkins) [reply src/shared/components/results/ResultsContainer.tsx:87]: Ok just add to the Jira 👍
